### ISSUE TITLE
Worked on RHEL UBI8 Image

### ISSUE THREAD

**Issue by kehundley-fadv** [cyberark/PwnKit-Hunter#1]
I just wanted to let you know that I tested this on a RHEL UBI8 docker image before I noticed the note about Debian and Ubuntu. It did actually work and alert me that the OS was vulnerable, and once I patched it, reported not vulnerable.

The only thing that was off was the output detail. The package name is `polkit` on RHEL, and the update instruction `yum install polkit` on RHEL.

Thanks, Ken

**Comment by kehundley-fadv** [comment]
Quick note, I only tested the C version. I didn't try running the python script

**Comment by AlonZa** [comment]
Hi,

I was not able to reproduce the behaviour. It gives a false-positive result.
Can you try it again and include versions of the packages and the output of the program?

**Comment by kehundley-fadv** [comment]
Sure, I'll try it again later today and document everything

**Comment by kehundley-fadv** [comment]
It may have been a false positive because polkit doesn't appear to be installed by default in the UBI base image I'm using.

Steps to reproduce:
1. docker run --rm -it -u root --entrypoint=/bin/bash registry.access.redhat.com/ubi8/dotnet-60
2. pkcheck --version > (not found)
3. yum install gcc git -y
4. Download and run your C script > "Your policykit-1 package is vulnerable"
5. yum install polkit -y
6. run your script > "Your policykit-1 package is up-to-date"
7. pkcheck --version > 0.115

Sry of the confusion, but everything matched my expectations. I ran the tool, it said I was vulnerable, I installed the latest package, it said I was patched.
